apps/examples/echo/rpmsg-echo.c

```diff
@@ -75,6 +75,10 @@ int app(struct rpmsg_device *rdev, void *priv)
 	}
 
 	LPRINTF("Successfully created rpmsg endpoint.\r\n");
+
+	LPRINTF("RPMsg device RX buffer size: %#x\r\n", rpmsg_virtio_get_tx_buffer_size(rdev));
```

Comment:
This line should say "TX buffer size".

Reply:
copy past without using my brain... thanks!